src/worker/batch/summary/download.py

```diff
@@ -10,15 +10,15 @@
 
 def get_nucleotide(run_id):
     file_key = f'{nucleotide_dir}/{run_id}{nucleotide_suffix}'
-    return get_file_contents('lovelywater', file_key)
+    return get_file_contents('lovelywater2', file_key)
 
 def get_protein(run_id):
     file_key = f'{protein_dir}/{run_id}{protein_suffix}'
-    return get_file_contents('lovelywater', file_key)
+    return get_file_contents('lovelywater2', file_key)
 
 def get_rdrp(run_id):
     file_key = f'{rdrp_dir}/{run_id}{rdrp_suffix}'
-    return get_file_contents('serratus-bio', file_key)
+    return get_file_contents('lovelywater2', file_key)
```

Comment:
this is intentional, the `serratus-bio` bucket doesn't have any of the  RdRP summary files for the  run_ids in the test